packages/webapp/components/layouts/RecruiterSelfServeLayout.tsx

```diff
@@ -13,7 +14,11 @@ const RecruiterSelfServeLayoutWithIntercom = ({
 }) => {
   useIntercom();
 
-  return <RecruiterLayout {...layoutProps}>{children}</RecruiterLayout>;
+  return (
+    <PendingSubmissionProvider>
```

Comment:
What do you actually use it for?

Reply:
When the user uploads or submits a link in the modal, we immediately transition them to analyze page. So we need to keep their input as context for the analyze page